### PR TITLE
PAR-538: Groundwork for beta flows

### PR DESCRIPTION
**Pull request by kalpaitch**

### CODE REVIEW

File: sync/par_flows.par_flow.partnership_direct.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+uuid: cfe7446a-c296-40ce-8cec-6c2879345924
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: GwqkAtbze34CwV4eRYXLoZ6cBEG0D7XmQ_xRp3_m-oA
+id: partnership_direct
+label: 'Direcr Partnership Flow'
```
Comment (stemount) [inline sync/par_flows.par_flow.partnership_direct.yml]: Direcr-gate

File: web/modules/features/par_partnership_flows/config/install/par_flows.par_flow.partnership_direct.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+status: true
+dependencies: {  }
+id: partnership_direct
+label: 'Direcr Partnership Flow'
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/config/install/par_flows.par_flow.partnership_direct.yml]: direcr-gate strikes again

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsBusinessSizeForm.php [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, ParDataPartnership $par_data_partnership = NULL) {
+    $this->retrieveEditableValues($par_data_partnership);
+
+    $form['info'] = [
+      '#markup' => t('Change the number of individual associations you represent.'),
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsBusinessSizeForm.php]: to keep this uniform, let's add `'#type' => 'markup'`

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsContactForm.php [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, ParDataPartnership $par_data_partnership = NULL, ParDataPerson $par_data_person = NULL) {
+    $this->retrieveEditableValues($par_data_partnership, $par_data_person);
+    $person_bundle = $this->getParDataManager()->getParBundleEntity('par_data_person');
+
+    // Leading paragraph.
+    $form['leading_paragraph'] = [
+      '#type' => 'markup',
+      '#markup' => t('<p><strong>Change the main contact for your organisation</strong></p>'),
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsContactForm.php]: this is fine I guess as it is markup, but `#prefix` and `#suffix` maybe tidier

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsSicCodeForm.php [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+    $form['intro'] = [
+      '#markup' => $this->t('Change the SIC Code of your business'),
+    ];
+
+
+    $form['sic_code'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('SIC Code'),
+//      '#default_value' => isset($par_data_organisation->get('trading_name')->getValue()[$trading_name_delta]) ? $par_data_organisation->get('trading_name')->getValue()[$trading_name_delta] : '',
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsSicCodeForm.php]: just checking if this is meant to be commented out?

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsSicCodeForm.php [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+//      '#description' => $this->t('Sometimes companies trade under a different name to their registered, legal name. This is known as a \'trading name\'. State any trading names used by the business.'),
+    ];
+
+    $form['next'] = [
+      '#type' => 'submit',
+      '#value' => t('Save'),
+    ];
+dump($this->getFlow());
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsSicCodeForm.php]: developer function.

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+    return parent::buildForm($form, $form_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validateForm(array &$form, FormStateInterface $form_state) {
+    // No validation yet.
+    parent::validateForm($form, $form_state);
+//    $premises = $this->getRouteParam('par_data_premises');
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php]: is this section supposed to be commented?

File: web/modules/features/par_partnership_flows/config/install/par_flows.par_flow.partnership_direct.yml (hunk 2 above)
Comment (kalpaitch) [reply web/modules/features/par_partnership_flows/config/install/par_flows.par_flow.partnership_direct.yml]: ha ha ha

**Comment by kalpaitch** [comment]
@stemount Don't worry about any of the stuff inside any of the buildForm, submit or validate methods this will all be re-done in separate tickets. Not sure whether it's best to leave it in as a starting point or work on it fresh from before... we've gone with the latter.

**Pull request merged**